source/configuring_domain/05_configuring_domain/regional_settings.rst

```diff
@@ -0,0 +1,20 @@
+Regional settings
+=================
+
+Domain timezone
+---------------
+
+.. image:: images/regional_settings.png
+
+Notification emails (reports and alerts) are sent according to the timezone of the domain. By default, the timezone of all OpenDataSoft domains is UTC.
```

Comment:
Well, existing domains have been set on Paris as default. Not sure if we want to explain that though

Reply:
I would be more precise about the effect of the timezone, something like:
`The time of events in report and alert emails are localized according to the timezone of the domain`
Otherwise it could make people think we actually send the emails at a specific time depending of the domain

Reply:
@vbarbaresi Yes I'm aware of that but I don't think we need to explain that in the documentation. If I'm correct, from now on all new domains will be set on UTC timezone by default, so I believe that it's better to stick with that simple explanation.